zipkin-server/src/main/resources/zipkin-server.yml

```diff
@@ -83,13 +83,18 @@ zipkin:
     # - .*example2.com
     # Default is "match all websites"
     instrumented: .*
+  server:
```

Comment:
I thought reusing the `zipkin.query` namespace would be appropriate for this property. @adriancole WDYT?